samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
Cookie persistence is a matter for the cookie middleware, and a line comment on a sample.

If you are asking about protection then yes, it has scope for crypto agility which in turn would support upgrades.